frontend/src/modules/editor/components/KeyboardShortcuts.js

```diff
@@ -0,0 +1,154 @@
+/* @flow */
+
+import * as React from 'react';
+import onClickOutside from 'react-onclickoutside';
+import { Localized } from 'fluent-react';
+
+import './KeyboardShortcuts.css';
+
+
+type Props = null;
+
+
+type State = {|
+    visible: boolean,
+|};
+
+
+/*
+ * Shows a list of keyboard shortcuts.
+ */
+export class EditorSettingsBase extends React.Component<Props, State> {
+    constructor() {
+        super();
+        this.state = {
+            visible: false,
+        };
+    }
+
+    toggleVisibility = () => {
+        this.setState({
+            visible: !this.state.visible,
+        });
+    }
+
+    // This method is called by the Higher-Order Component `onClickOutside`
+    // when a user clicks outside the keyboard shortcut overlay.
+    handleClickOutside = () => {
+        this.setState({
+            visible: false,
+        });
+    }
+
+    render() {
+        return <div className="keyboard-shortcuts">
+            <Localized
+                id="keyboard-shortcuts-button"
+                attrs={{ title: true }}
+            >
+                <div
+                    className="selector far fa-keyboard"
+                    title="Keyboard Shortcuts"
+                    onClick={ this.toggleVisibility }
+                />
+            </Localized>
+
+            { !this.state.visible ? null :
+            <div
+                className="overlay"
+                onClick={ this.toggleVisibility }
+            >
+                <Localized id="keyboard-shortcuts-overlay-title">
+                    <h2>Keyboard Shortcuts</h2>
```

Comment:
Iirc @stasm recommended that strings be put in quotes when they are localizable, but I can't remember why... :)

Here that would be `<h2>{ 'Keyboard Shortcuts' }</h2>` for example.

(Once again that might be something that I didn't do elsewhere in the code base, sorry about that. :/ )

Reply:
When the whole contents of the element is simple text, it's fine to use both approaches.

The recommendation to use the string literal in quotes comes from my anticipating that in the future we'll want to extract these source strings automatically. When this happens, it will be important that they are all strings (`<p>{"I'm a string with a Fluent {$variable}."}</p>`) rather than a list of React nodes (`<p>I'm a React node with {things.map(item => item.toString())}.</p>`). In the former case, extraction is trivial. In the latter, not so much.

Reply:
So I'm keeping that unchanged.